pnpm-workspace.yaml

```diff
@@ -1,3 +1,3 @@
 packages:
   - "docs"
-
```

Comment:
I'd like to target a root `packages` folder with every individual packages. I understand that this emulator is a package of his own, could you move it outside of `src` ?
In fine, the whole `src` folder will disappear but I don't think this should be part of this already mega long PR 😅

Reply:
I agree, it felt weird to put it in src, I have the same issue with the create-skybridge command

Reply:
Target is everything in `packages` without any remaining root `src` just like next.js repo